TFLite/src/init.py

```diff
@@ -27,4 +22,9 @@ def init_tflite_files():
     )
 
 if __name__ == "__main__":
-    init_tflite_files()
\ No newline at end of file
+
+    # Path to schema and path where the directory should be generated
+    schema_path = "./schema.fbs"
```

Comment:
Make this a global constants? SCHEMA_PATH and OUT_PATH?

Reply:
Made naming change to constants